dev/com.ibm.ws.ejbcontainer.core/src/com/ibm/wsspi/ejbcontainer/JITDeploy.java

```diff
@@ -80,41 +81,27 @@ private JITDeploy()
      * @return the compatibility flags
      * @see #isRMICCompatibleValues
      */
-    public static int parseRMICCompatible(String options) // PM46698
-    {
+    public static int parseRMICCompatible(String options) {
         final boolean isTraceOn = TraceComponent.isAnyTracingEnabled();
         if (isTraceOn && tc.isEntryEnabled())
             Tr.entry(tc, "parseRMICCompatible: " + options);
 
         int flags;
 
-        if (options == null)
-        {
+        if (options == null) {
             flags = RMIC_COMPATIBLE_DEFAULT;
-        }
-        else if (options.equals("none"))
-        {
+        } else if (options.equals("none")) {
             flags = 0;
```

Comment:
Should this and the next two cases use the constants for consistency in the code?

Reply:
Existing code that wasn't using constants... but that is because constants didn't exist previously.  Since I had added a new constant for "all", then for consistency, I added another constant for "none" and set the default constant to the new "none" constant and updated the code to use the constants. Constants are now used consistently and support the ability to change the "default" value without changing the rest of the code (not expecting to ever do that).